highfidelity/hifi_tests
# Added model protocol test.

# Description
Adds a test of the "model" entity properties protocol.
# Testing
1.  Run Interface
2.  Load the following test: https://github.com/NissimHadar/hifi_tests/blob/addModelProtocolTest/tests/protocol/model/test.js
3.  Verify that test passes (green box)
